include/RED4ext/Scripting/Natives/Vector2.hpp

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include <RED4ext/Common.hpp>
+#include <cstdint>
+
+namespace RED4ext
+{
+struct Vector2
+{
+    static constexpr const char* NAME = "Vector2";
+    static constexpr const char* ALIAS = NAME;
+
+    Vector2()
+        : X(0)
+        , Y(0)
+    {
+    }
+
+    Vector2(float x, float y)
```

Comment:
```suggestion
    Vector2(float aX, float aY)
```

Do the same for the other.